tests/utils_test.py

```diff
@@ -176,7 +176,7 @@ def load_and_clean_json(location):
     """
     with io.open(location, encoding='utf-8') as l:
         content = l.read()
-    data = json.loads(content)
+    data = {'Document': json.loads(content)}
```

Comment:
json files don't start with `Document`. Is this some kind of workaround? The same goes for the yaml version below